code/game/objects/structures/window.dm

```diff
@@ -309,10 +318,29 @@ var/list/one_way_windows
 			one_way_windows = list()
 		one_way_windows.Add(src)
 		P.use(1)
-		to_chat(user, "<span class='notice'>You place a sheet of plastic over the window.</span>")
+		to_chat(user, "<span class='notice'>You place a sheet of plastic over the [src].</span>")
 		overlays += oneway_overlay
 		return
 
+	
+	if(istype(W, /obj/item/stack/light_w))
+		var/obj/item/stack/light_w/LT = W
+		if (smartwindow)
+			to_chat(user, "<span class='notice'>This [src] already has electronics in it.</span>")
```

Comment:
This the window

Reply:
there's no \this right

Reply:
\the, \The // insert 'the' (if needed)
\a, \an,   // insert 'a(n)' or 'some' (if needed)
\A, \An

http://www.byond.com/docs/ref/info.html#/DM/text/macros